clang/docs/ReleaseNotes.rst

```diff
@@ -249,6 +249,12 @@ Bug Fixes in This Version
 
 - Clang now doesn't produce false-positive warning `-Wconstant-logical-operand`
   for logical operators in C23.
+  Fixes (`#64356 <https://github.com/llvm/llvm-project/issues/64356>`_).
+- ``__is_trivially_relocatable`` no longer returns ``true`` for non-object types
+  such as references and functions, and no longer returns ``false`` for volatile-qualified types.
+  Fixes (`#67498 <https://github.com/llvm/llvm-project/issues/67498>`_) and
+  (`#77091 <https://github.com/llvm/llvm-project/issues/77091>`_)
+
```

Comment:
That's should be moved _after_ the next line (and you can use the same `#GHXXXX` syntax )